prompts/github.go

```diff
@@ -2,76 +2,171 @@ package prompts
+		secrets[config.SpeakeasyApiKey] = formatSecretName(defaultSpeakeasyAPIKeySecretName)
+		githubWorkflow = &config.GenerateWorkflow{
+			Name: "Generate",
+			On: config.GenerateOn{
+				WorkflowDispatch: config.WorkflowDispatch{
+					Inputs: config.Inputs{
+						Force: config.Force{
+							Description: "Force generation of SDKs",
+							Type:        "boolean",
+							Default:     false,
+						},
 					},
 				},
-			},
-			Schedule: []config.Schedule{
-				{
-					Cron: "0 0 * * *",
+				Schedule: []config.Schedule{
+					{
+						Cron: "0 0 * * *",
+					},
 				},
 			},
-		},
-		Jobs: config.Jobs{
-			Generate: config.Job{
-				Uses: "speakeasy-api/sdk-generation-action/.github/workflows/sdk-generation.yaml@v14",
-				With: map[string]any{
-					"speakeasy_version": "latest",
-					"force":             "${{ github.event.inputs.force }}",
-					config.OpenAPIDocs:  fmt.Sprintf("- %s\n", quickstart.WorkflowFile.Sources[specName].Inputs[0].Location),
-					config.Mode:         "pr",
-					config.Languages:    fmt.Sprintf("- %s\n", targetType),
+			Jobs: config.Jobs{
+				Generate: config.Job{
+					Uses: "speakeasy-api/sdk-generation-action/.github/workflows/sdk-generation.yaml@v15",
+					With: map[string]any{
+						"speakeasy_version": "latest",
+						"force":             "${{ github.event.inputs.force }}",
+						config.Mode:         "pr",
+					},
+					Secrets: secrets,
 				},
-				Secrets: secrets,
 			},
-		},
-		Permissions: config.Permissions{
-			Checks:       config.GithubWritePermission,
-			Statuses:     config.GithubWritePermission,
-			Contents:     config.GithubWritePermission,
-			PullRequests: config.GithubWritePermission,
-		},
+			Permissions: config.Permissions{
+				Checks:       config.GithubWritePermission,
+				Statuses:     config.GithubWritePermission,
+				Contents:     config.GithubWritePermission,
+				PullRequests: config.GithubWritePermission,
+			},
+		}
+	}
+
+	secrets := githubWorkflow.Jobs.Generate.Secrets
+	for _, source := range workflow.Sources {
+		for _, input := range source.Inputs {
+			if input.Auth != nil {
+				secrets[input.Auth.Secret] = formatSecretName(strings.ToLower(input.Auth.Secret))
+			}
+		}
+
+		for _, overlay := range source.Overlays {
+			if overlay.Auth != nil {
+				secrets[overlay.Auth.Secret] = formatSecretName(strings.ToLower(overlay.Auth.Secret))
+			}
+		}
 	}
+	githubWorkflow.Jobs.Generate.Secrets = secrets
+	mode := githubWorkflow.Jobs.Generate.With[config.Mode].(string)
 
-	if quickstart.WorkflowFile.Sources[specName].Inputs[0].Auth != nil {
-		genWorkflow.Jobs.Generate.With[config.OpenAPIDocAuthHeader] = quickstart.WorkflowFile.Sources[specName].Inputs[0].Auth.Header
-		genWorkflow.Jobs.Generate.Secrets[config.OpenAPIDocAuthToken] = formatSecretName(quickstart.WorkflowFile.Sources[specName].Inputs[0].Auth.Secret)
+	modeOptions := []huh.Option[string]{
+		huh.NewOption("pr mode: creates a running PR that you can merge at your convenience [RECOMMENDED]", "pr"),
```

Comment:
Does it work to add "inline" styling here? It's a bit hard to read. Would be nice to make `PR Mode` and `Direct Mode` capitalized and bold, possibly even a different color

Reply:
I can try it and see what happens

Reply:
It worked